openlane/steps/openroad.py

```diff
@@ -715,9 +720,28 @@ class Floorplan(OpenROADStep):
         ),
     ]
 
+    class Mode(str, Enum):
```

Comment:
Feel like an enum is overkill here instead of `Literal['template', 'absolute', 'relative']`

Reply:
I think it is better if we decide to change the internal name (value of the properties) later.